build.txt

```diff
@@ -1,6 +1,6 @@
 displayName = Wireless
 author = gardenapple
-version = 1.5.3
+version = 1.5.10
```

Comment:
Sorry, I was not paying attention to your updates: were there actually 7 updates here or do you increment the version with each commit?

Reply:
I have been increasing it every bugfix commit because it can't the version on steam isn't 1.5.10. Otherwise I cant publish it.